war/src/main/js/api/plugins.js

```diff
@@ -52,6 +55,14 @@ exports.availablePlugins = [
         ]
     },
     {
+        "category":"Pipelines and Continuous Delivery",
+        "plugins": [
+            { "name": "workflow-aggregator" },
+            { "name": "github-branch-source" },
+            { "name": "workflow-multibranch" }
```

Comment:
@oleg-nenashev See item three in my initial comment. This discussion is still ongoing. Note for example also the lack of Git Plugin or Email Ext here.

FWIW in my current local draft this category also includes Build Pipeline, Delivery Pipeline, Copy Artifact, Parameterized Trigger, and – this is probably weird – Throttle Concurrent.